nomad/grapl-core.nomad

```diff
@@ -1174,10 +1128,9 @@ job "grapl-core" {
         PLUGIN_REGISTRY_KERNEL_ARTIFACT_URL             = var.plugin_registry_kernel_artifact_url
         PLUGIN_REGISTRY_ROOTFS_ARTIFACT_URL             = var.plugin_registry_rootfs_artifact_url
         PLUGIN_REGISTRY_HAX_DOCKER_PLUGIN_RUNTIME_IMAGE = var.container_images["hax-docker-plugin-runtime"]
-        # Plugin Execution code/image doesn't exist yet; change this once it does!
-        PLUGIN_EXECUTION_CONTAINER_IMAGE      = "grapl/plugin-execution-sidecar-TODO"
-        PLUGIN_REGISTRY_BUCKET_AWS_ACCOUNT_ID = var.plugin_registry_bucket_aws_account_id
-        PLUGIN_REGISTRY_BUCKET_NAME           = var.plugin_registry_bucket_name
+        PLUGIN_EXECUTION_IMAGE                          = var.container_images["generator-executor"] # TODO: add support for analyzer too
```

Comment:
I suspect what we'll end up doing is changing this to a `plugin-executor` that can make both GetExecuteAnalyzer and GetExecuteGenerator requests; but this PR is too big already IMO!